README.md

```diff
@@ -1,53 +1,57 @@
+[![Tests and Linter](https://github.com/tolstoyevsky/cusdeb-web-client/actions/workflows/checks.yml/badge.svg)](https://github.com/tolstoyevsky/cusdeb-web-client/actions/workflows/checks.yml/badge.svg)
+[![Website cusdeb.com](https://img.shields.io/website-up-down-green-red/https/lbesson.bitbucket.io.svg)](https://cusdeb.com)
+
 # CusDeb Web Client
 
-CusDeb Web Client is a web client which relies on such CusDeb services as [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api), [Black Magic](https://github.com/tolstoyevsky/blackmagic), [Dominion](https://github.com/tolstoyevsky/dominion) and [Orion](https://github.com/tolstoyevsky/orion).
+CusDeb Web Client exposes the entirety of CusDeb's functionality to the user, including customizing OS images for wide variety of single-board computers.
```

Comment:
CusDeb Web Client exposes the whole [CusDeb](https://cusdeb.com)'s functionality to users, including customizing OS images for wide variety of single-board computers.

Reply:
![cusdeb2](https://user-images.githubusercontent.com/5358223/112978949-9eada480-9160-11eb-9480-1d1c7548593e.gif)